build.sbt

```diff
@@ -1,45 +1,22 @@
-import ReleaseTransformations._
-import sbtrelease.ReleasePlugin.autoImport.releaseStepCommand
```

Comment:
Removed a lot of release-related stuff in the build file, since it's likely no longer needed